examples/2d/texture_atlas.rs

```diff
@@ -1,11 +1,17 @@
-//! In this example we generate a new texture atlas (sprite sheet) from a folder containing
+//! In this example we generate 4 texture atlases (sprite sheets) from a folder containing
```

Comment:
```suggestion
//! In this example we generate four texture atlases (sprite sheets) from a folder containing
```

Word number seems more readable here